api/Permissions.json

```
@@ -223,6 +223,39 @@
           }
         }
       },
+      "permission_captured-surface-control": {
+        "__compat": {
+          "description": "`captured-surface-control` permission",
```

Let's link to the spec:

```suggestion
          "description": "`captured-surface-control` permission",
          "spec_url": "https://w3c.github.io/mediacapture-surface-control/#dfn-captured-surface-control",
```